docs/content/en/docs-dev/operator-manual/piped/configuration-reference.md

```diff
@@ -34,6 +34,7 @@ spec:
 | eventWatcher | [EventWatcher](/docs/operator-manual/piped/configuration-reference/#eventwatcher) | Optional Event watcher settings. | No |
 | secretManagement | [SecretManagement](/docs/operator-manual/piped/configuration-reference/#secretmanagement) | The using secret management method. | No |
 | notifications | [Notifications](/docs/operator-manual/piped/configuration-reference/#notifications) | Sending notifications to Slack, Webhook... | No |
+| appSelector | map[string]string | List of labels to filter all applications this piped will handle. Currently, it is only be used to filter the applications suggested for adding from the control plane. | No |
```

Comment:
not really related to this pull but I just realized that this is `List of ...` so should we change the name as `appSelectors`?

Reply:
This is based on [k8s selector](https://kubernetes.io/docs/concepts/services-networking/service/#defining-a-service)
But it seems good to combine `appSelectors` with `labels`. wdyt?

Reply:
Then it will be `appLabelsSelector` right? 😄

Reply:
on the second thought, I guess `labels` is unnecessary, so I think we should keep it as is. Sorry for the interruption 🙏

Reply:
Umm.., It took me a while to think `appSelector` is okay.
Do you feel a bit strange?

Reply:
Yep, so lets keep it as is, thank you 🙌